Composer/packages/client/src/components/WebChat/utils/conversationService.ts

```diff
@@ -277,7 +276,7 @@ export class ConversationService {
       });
     } catch (ex) {
       const response: AxiosResponse = ex.response;
-      const err: DirectLineLog = {
+      const err = {
```

Comment:
Isn't err still of the type DirectlineLog

Reply:
I got rid of the `DirectlineLog` type